src/main/kotlin/no/nav/Consumers.kt

```diff
@@ -26,7 +27,8 @@ interface Consumers {
     val pdlClient: PdlClient
     val safClient: SafClient
     val digdirClient: DigdirClient
-    val utbetalinger: UtbetalingerClient
+    val utbetalingerClient: UtbetalingerClient
```

Comment:
Hva er grunnen til at vi beholder soap-klienten?

Reply:
Tanken min var å heller fjerne det i ettertid når vi hadde fått testet at rest-apiet fungerte, men det har vel egentlig ikke noe særlig for seg å gjøre det på den måten 🤷🏼‍♂️